nautobot/dcim/fields.py

```diff
@@ -104,3 +105,16 @@ def __init__(self, **kwargs):
 
 
 PathField.register_lookup(PathContains)
+
+
+class JSONPathField(JSONArrayField):
```

Comment:
Is there any use-case for the old `PathField` once this is replaced? Couldn't we just go ahead and nuke the old one in exchange for this replacement and call this one `PathField`?